ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -61,6 +61,39 @@ def msort(a):
     return ivy.msort(a)
 
 
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {
+        "0.4.13 and below": (
+            "float16",
+            "bfloat16",
+        )
+    },
+    "jax",
+)
+def argpartition(
+    a,
+    kth,
+    axis=-1,
+    kind="stable",
+    order=None,
+) -> ivy.Array:
```

Comment:
We used the decorator `@to_ivy_arrays_and_back`, which means that the return type of the function should be a frontend array, not `ivy.Array`. It's best not to typehint the frontend implementations.